client/my-sites/plans/ecommerce-trial/index.tsx

```diff
@@ -0,0 +1,314 @@
+import { getPlans, PLAN_ECOMMERCE, PLAN_ECOMMERCE_MONTHLY } from '@automattic/calypso-products';
+import { Button, Card } from '@automattic/components';
+import { getCurrencyObject } from '@automattic/format-currency';
+import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import BodySectionCssClass from 'calypso/layout/body-section-css-class';
+import { getPlanRawPrice, getPlan } from 'calypso/state/plans/selectors';
+import ECommerceTrialBanner from './ecommerce-trial-banner';
+import TrialFeatureCard from './trial-feature-card';
+
+import './style.scss';
+
+const ECommerceTrialPlansPage = () => {
+	const translate = useTranslate();
+
+	const eCommercePlanAnnual = getPlans()[ PLAN_ECOMMERCE ];
+	const eCommercePlanMonthly = getPlans()[ PLAN_ECOMMERCE_MONTHLY ];
+
+	const eCommercePlanPrices = useSelector( ( state ) => ( {
+		annualPlanPrice: getPlanRawPrice( state, eCommercePlanAnnual.getProductId(), false ) || 0,
+		annualPlanMonthlyPrice: getPlanRawPrice( state, eCommercePlanAnnual.getProductId(), true ) || 0,
+		monthlyPlanPrice: getPlanRawPrice( state, eCommercePlanMonthly.getProductId() ) || 0,
+		currencyCode: getPlan( state, eCommercePlanAnnual.getProductId() )?.currency_code || '',
+	} ) );
+
+	const { symbol: currencySymbol } = getCurrencyObject( 0, eCommercePlanPrices.currencyCode );
+
+	// Defaulting to annual while we don't add the toggle
+	const isAnnualSubscription = true;
+
+	const percentageSavings = Math.floor(
+		( 1 - eCommercePlanPrices.annualPlanMonthlyPrice / eCommercePlanPrices.monthlyPlanPrice ) * 100
+	);
+
+	const features = [
+		{
+			expanded: true,
+			illustration: '/calypso/images/plans/wpcom/ecommerce-trial/general-features.png',
```

Comment:
We're phasing out the `/static/images` folder according to [the directory's README](https://github.com/Automattic/wp-calypso/tree/trunk/static/images). We should move these into `/client/assets/images`, that way we can take advantage of webpack. Example:

```
// Import images at the top
import generalFeatures from 'calypso/assets/images/ecommerce-trial/general-features.png';
```

Then:

`illustration: generalFeatures,`